src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,92 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (movie: Movie) => void,
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, addCount] = useState(1);
+  const [state, setState] = useState({
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  });
+
+  const handlerStateFilm = (key: string, value: string) => {
+    setState({ ...state, [key]: value });
+  };
+
+  // eslint-disable-next-line max-len
+  const patternUrl = /^((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=+$,\w]+@)?[A-Za-z0-9.-]+|(?:www\.|[-;:&=+$,\w]+@)[A-Za-z0-9.-]+)((?:\/[+~%/.\w-_]*)?\??(?:[-+=&;%@,.\w_]*)#?(?:[,.!/\\\w]*))?)$/;
+  // eslint-disable-next-line max-len
+  const patternLetterAndNumbers = /^([a-zA-Z0-9\u0600-\u06FF\u0660-\u0669\u06F0-\u06F9 _.-]+)$/;
+  const error = patternLetterAndNumbers.test(state.title.trim())
```

Comment:
why does patternLetterAndNumbers check if title not empty?
```suggestion
  const error = state.title.trim() && state.imdbId.trim() ...
```